model/src/w3iopomd.F90

```diff
@@ -1349,6 +1353,15 @@ SUBROUTINE W3IOPON_READ(IOTST, IMOD_IN, filename, ncerr)
       IGRD = 1
     END IF
 
+    ! Determine if we are reading a per-time-step file
+    per_time_step = PRESENT(TOUT)
+    IF (per_time_step) THEN
+      WRITE(TIMETAG, '(I8.8, ".", I6.6)') TOUT(1), TOUT(2)
+      filename = TRIM(FNMPRE) // TRIM(TIMETAG) // '.out_pnt.' // TRIM(FILEXT) // '.nc'
+    ELSE
+      filename = FNMPRE(:LEN_TRIM(FNMPRE))//'out_pnt.'//FILEXT(:LEN_TRIM(FILEXT))//'.nc'
+    END IF
+
```

Comment:
@AliS-Noaa  - Particularly because the code will fail without a very informative error, before trying to open the file, adding in a check to see if the file exists and if it does not, providing an informative error statement before existing would be helpful.  Please let me know if you need any assistance adding this.